server/src/test/scala/cromwell/engine/workflow/lifecycle/MaterializeWorkflowDescriptorActorSpec.scala

```diff
@@ -126,7 +126,7 @@ class MaterializeWorkflowDescriptorActorSpec extends CromwellTestKitWordSpec wit
             wfDesc.knownValues.head._2 shouldBe WomInteger(5)
             wfDesc.getWorkflowOption(WorkflowOptions.WriteToCache) shouldBe None
             wfDesc.getWorkflowOption(WorkflowOptions.ReadFromCache) shouldBe None
-            wfDesc.pathBuilders.size shouldBe 1
+            wfDesc.pathBuilders.size shouldBe 2 // // one each for the local and http filesystems
```

Comment:
`2 // // one` should be `2 // one`

Reply:
To avoid travis re-runs, I'll address this is my part II, (stable(r) names for HTTP inputs)